### ISSUE TITLE
Xcode warning after last update

### ISSUE THREAD

**Issue by MincDev** [evgenyneu/Auk#81]
* Xcode version. 10.1.

After the last xcode update I am getting the following warning:

_Scrollable content size is ambiguous for "Slideshow"._   

The code still works as it always has and no issues are present on devices, however the warning is quite irritating since I use this slideshow on multiple screens.

Any ideas on a fix?

**Comment by evgenyneu** [comment]
Thanks for reporting the issue, @MincDev. I've created a demo app that shows two images, but I do not see this warning in Xcode 10.1. Is the demo working for you as well?

[AukOneImageDemo.zip](https://github.com/evgenyneu/Auk/files/2573960/AukOneImageDemo.zip)

**Comment by MincDev** [comment]
Hi @evgenyneu, Thank you for your time in assisting me. I opened your project and also saw that the warning is gone. I tried resizing the scrollview so it is displayed the same way as in my app, but the warning is still not there. I am not sure what might be causing this on my app, but this is clearly not an issue with your slideshow. Apologies for wasting your time and thanks again for your assistance.
